test/HydratorAwareTraitTest.php

```diff
@@ -28,6 +29,7 @@ public function testSetHydrator(): void
 
     public function testGetHydrator(): void
     {
+        /** @psalm-suppress InvalidScalarArgument False positive */
```

Comment:
Why are we suppressing this? Isn't `HydratorAwareTrait` a valid trait? :thinking:

Reply:
See above.